erpc.yaml

```diff
@@ -26,377 +34,126 @@ projects:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 8453
-        failsafe:
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 10
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 1284
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 1285
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
+      allowedMethods:
+        - 'GET'
+        - 'POST'
+        - 'OPTIONS'
+
+    networkDefaults:
+      failsafe:
+        circuitBreaker: ~
+        timeout:
+          duration: 60s
+        retry:
+          maxAttempts: 3
+          delay: 500ms
+          backoffMaxDelay: 10s
+          backoffFactor: 0.3
+          jitter: 200ms
+        hedge:
+          delay: 1000ms
+          maxCount: 2
+
+    upstreamDefaults:
+      jsonRpc:
+        supportsBatch: true
+        batchMaxWait: 100ms
+        batchMaxSize: 100
+      failsafe:
+        timeout:
+          duration: 15s
+        retry:
+          maxAttempts: 2
+          delay: 1000ms
+          backoffMaxDelay: 10s
+          backoffFactor: 0.3
+          jitter: 500ms
+
     upstreams:
-      #- id: alchemy-global
-      #  endpoint: ${ALCHEMY_ENDPOINT}
-      #  jsonRpc:
-      #    supportsBatch: false
-      #  rateLimitBudget: global-alchemy
-      #  failsafe:
-      #    timeout:
-      #      duration: 15s
-      #    retry:
-      #      maxCount: 2
-      #      delay: 1000ms
-      #      backoffMaxDelay: 10s
-      #      backoffFactor: 0.3
-      #      jitter: 500ms
       - id: cdp-chain-8453
         type: evm
-        endpoint: ${CDP_BASE_RPC_URL}
-        jsonRpc:
-          supportsBatch: true
-          batchMaxWait: 100ms
-          batchMaxSize: 100
+        endpoint: https://${CDP_BASE_RPC_URL}
```

Comment:
make sure to set `CDP_BASE_RPC_URL ` in `.env` without `https://`

Reply:
Ok, will do